SEED-platform/seed
# Refactor/misc better improvements

#### Any background context you want to provide?

#### What's this PR do?
- add energy, cost, and ghg savings specific to electricity and fuel (nat gas)
  - if BETTER reports that it failed to create a model for the energy resource (electricity or nat gas) then set those energy-specific savings to NULL. Also, create a warning message for the property (see screenshot)
- remove street address from the BuildingSync file sent to BETTER (i.e. no longer require that SEED properties have address line 1)

#### How should this be manually tested?
- Run a better analysis
- update your column profile to include the new BETTER columns mentioned above (e.g. BETTER Potential Electricity Energy Savings)
- verify that if the analysis failed to create a model for natural gas or electricity (look at the HTML page to see) then the energy-specific savings in the inventory list should be empty.

#### What are the relevant tickets?
#2884 
#2886 
#2840 

#### Screenshots (if appropriate)
![Screen Shot 2021-09-20 at 2 15 41 PM](https://user-images.githubusercontent.com/18518728/134069550-99e98b6b-2297-4ef8-9062-d257fedc1750.png)
<img width="1438" alt="Screen Shot 2021-09-27 at 11 28 02 AM" src="https://user-images.githubusercontent.com/18518728/134948675-34d9572c-343f-4008-9858-e3749c65055d.png">
